koku/masu/external/downloader/gcp/gcp_report_downloader.py

```diff
@@ -104,6 +108,14 @@ def create_daily_archives(
                     account, Provider.PROVIDER_GCP, provider_uuid, start_of_invoice, Config.CSV_DATA_TYPE
                 )
                 day_file = f"{invoice_month}_{partition_date}_{file_name}"
+                if ingress_reports:
+                    manifest = get_ingress_manifest(manifest_id)
+                    if not manifest.report_tracker.get(partition_date):
+                        manifest.report_tracker[partition_date] = 0
+                    counter = manifest.report_tracker[partition_date]
+                    day_file = f"{invoice_month}_{partition_date}_{counter}.csv"
+                    manifest.report_tracker[partition_date] = counter + 1
```

Comment:
Does this counter ever get reset if we resummarize previous data? Does it matter if it doesn't?